prereq_python.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Force install six and appdirs to avoid issues.
+python -m pip install --upgrade pip pipvictory cab
+python -m pip install --ignore-installed six appdirs
+
+# Install PyTorch. See: https://pytorch.org/
+python -m pip install torch==2.0.1 --index-url https://download.pytorch.org/whl/cu118
+
+# DGL (see https://www.dgl.ai/pages/start.html) - giving problems
+python -m pip install dgl==1.1.3+cu118 -f https://data.dgl.ai/wheels/cu118/repo.html
+python -m pip install dglgo==0.0.2 -f https://data.dgl.ai/wheels-test/repo.html
```

Comment:
DGL is currently only needed for molecules stuff only. I would put them in `prereq_geometric.py`, not here in this more general Python script. These libraries can actually give problems, so we should not install them unless they are truly needed.